### PR TITLE
create stars section

### PR DESCRIPTION
**Pull request by buffi4ka**
https://buffi4ka.github.io/layout_stars/
https://buffi4ka.github.io/layout_stars/report/html_report/

I've been sitting for three hours and can't pass the automated tests. It's a sad statistic. Please help and point out my mistakes.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="star-section">
+      <div class="container">
```
Comment (polosanya) [inline src/index.html:12]: Do you really need a container here?

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+/* .star-section {
+  display: flex;
+  width: 140px;
+  height: 160px;
+  padding: 32px 22px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+} */
```
Comment (polosanya) [inline src/style.css:15]: Don't forget to remove the comments

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 96px;
```
Comment (polosanya) [inline src/style.css:21]: image

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  margin-right: 4px;
+}
+
+.stars {
+  list-style: none;
+  width: 96px;
```
Comment (polosanya) [inline src/style.css:27]: The styles are almost the same as for container. You shouldn't set the fixed sizes for it. Set the sizes and margins for stars elements

**Review by polosanya — CHANGES_REQUESTED** [review]
Nice! Let's try to pass the tests)
If you have any difficulties, please feel free to ask for help in qna chat or sign up for QnA session 🙂

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```
Comment (superpooperxxx) [inline src/style.css:4]: Never reset `margins` and `padding` with *. It takes so much resources. Use `*` only for `box-sizing` properties.

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (superpooperxxx) [inline src/style.css:10]: Delete what you don't use

**Review by superpooperxxx — CHANGES_REQUESTED** [review]
Hey, seems like you forgot to update gh-pages and tests link.😢 Cannot see the changes

**Review by superpooperxxx — CHANGES_REQUESTED** [review]
Hey, you need to create `develop` branch and create a Pull Request from it to `master`
❌
image
✅
image

**Review by superpooperxxx — CHANGES_REQUESTED** [review]
image

**Pull request closed without merging**